src/games/strategy/engine/framework/mapDownload/InstallMapDialog.java

```diff
@@ -280,16 +282,14 @@ private boolean replaceOldQuestion(final String message) {
   }
 
   private void install(final DownloadFileDescription selected, final int count, final int total) {
-    // get the destination file
-    final File destination = new File(GameRunner2.getUserMapsFolder(), selected.getMapName() + ".zip");
```

Comment:
Combine the comment of line 283 with line 284, make the variable name more descriptive, then update the reference on 286.